lib/blockchains/AlgorandBlockchain.js

```diff
@@ -270,8 +296,96 @@ class AlgorandBlockchain {
     }
   }
 
-  async disableWallet(_withdrawalAddresses, _hotWallet) {
-    console.error("Disable wallet for Algorand is not implemented")
+  async sendWithdrawalCoinsTransaction(coinAddress, totalAmount, hotWallet) {
+    await this.connect()
+
+    var chainTransaction = await this.chain.new.Transaction()
+    var txn = this.transferAlgoTx(hotWallet.address, coinAddress, totalAmount)
+
+    try {
+      var action = await this.chain.composeAction(chainjs.ModelsAlgorand.AlgorandChainActionType.Payment, txn)
+      chainTransaction.actions = [action]
+      const mAlgoFee = await chainTransaction.getSuggestedFee(chainjs.Models.TxExecutionPriority.Fast)
+
+      // Algorand wallet minimal balance formula https://developer.algorand.org/docs/get-details/dapps/smart-contracts/apps/#minimum-balance-requirement-for-a-smart-contract
+      // For the NOTE it should be 999750 micro algo
+      const amountWithoutFee = totalAmount - mAlgoFee - 1000000
+      chainTransaction = this.chain.new.Transaction()
+      txn = this.transferAlgoTx(hotWallet.address, coinAddress, amountWithoutFee)
+      action = await this.chain.composeAction(chainjs.ModelsAlgorand.AlgorandChainActionType.Payment, txn)
+      chainTransaction.actions = [action]
+      await chainTransaction.setDesiredFee(mAlgoFee)
+      await chainTransaction.prepareToBeSigned()
+      await chainTransaction.validate()
+      await chainTransaction.sign([chainjs.HelpersAlgorand.toAlgorandPrivateKey(hotWallet.privateKey)])
+      const tx_result = await chainTransaction.send(chainjs.Models.ConfirmType.After001)
```

Comment:
There seem to be a lot of repetitive code.
Why can't we reuse the `sendTransction` method?

Reply:
`sendTransction` locked on AppNoOp transaction type for now.
But we can change it in Algorand grouped transactions PR